pykep/core.cpp

```diff
@@ -190,7 +190,9 @@ PYBIND11_MODULE(core, m)
         py::pickle(&pykep::pickle_getstate_wrapper<kep3::planet>, &pykep::pickle_setstate_wrapper<kep3::planet>));
     // Planet methods.
     planet_class.def(
-        "eph", [](const kep3::planet &pl, double ep) { return pl.eph(ep); }, py::arg("ep"));
+        "eph", [](const kep3::planet &pl, double mjd2000) { return pl.eph(mjd2000); }, py::arg("mjd2000"));
```

Comment:
Just FYI, you can also write a single overload for ``eph`` with the help of ``std::variant``:

```c++
    planet_class.def(
        "eph", [](const kep3::planet &pl, const std::variant<double, kep3::epoch> &when)
        {
              return std::visit([&](const auto &v){return pl.eph(v);}, when);
        }, py::arg("when"));
```

One advantage of this approach is that pybind11 knows about ``std::variant`` and will generate a single, nicely-formatted docstring with the correct type annotation.

Reply:
what would the default value be for the arg though?

Reply:
for example in methods like elements we now have:
```c++
    planet_class.def(
        "period", [](const kep3::planet &pl, double mjd2000) { return pl.period(mjd2000); },
        py::arg("mjd2000") = 0., pykep::planet_period_docstring().c_str());
    planet_class.def(
        "period", [](const kep3::planet &pl, const kep3::epoch &ep) { return pl.period(ep); },
        py::arg("ep") = kep3::epoch{});
```

Reply:
you can use either ``py::arg("ep") = kep3::epoch{}`` or ``py::arg("ep") = 0.``, both should work as they are both convertible to ``std::variant<double, kep3::epoch>``.

Reply:
ok understood.... the second default value is actually not used at all, if not wrong. will remove and use variant